opwen_email_server/utils/email_parser.py

```diff
@@ -80,3 +83,28 @@ def _get_recipients(email: dict) -> Iterable[str]:
 def get_domains(email: dict) -> Iterable[str]:
     return frozenset(address.split('@')[-1]
                      for address in _get_recipients(email))
+
+
+def _get_as_base64(image_url):
+    res = requests.get(image_url)
+
+    if res.status_code != 200:
+        res.raise_for_status()
```

Comment:
`raise_for_status` already does not raise an exception if the status was okay, so you don't need the if statement here.